docs/source/index.rst

```diff
@@ -31,6 +31,7 @@ Contents
    serialization
    searching
    customization
+   conventions
 
 .. include:: ../../README.rst
 
```

Comment:
Oh and please check your docs, currently this file here is cutting sentences ! This is also not coll and really hard for people who use for example translation tools.

Rule of thumb:

One sentence per line, use line breaks which makes sense, line should not longer than around 130.

Please do not use pep8 or so on docs !

:)